src/ui/zcl_abapgit_services_repo.clas.abap

```diff
@@ -206,12 +198,15 @@ CLASS zcl_abapgit_services_repo IMPLEMENTATION.
       iv_ign_subpkg       = is_repo_params-ignore_subpackages
       iv_master_lang_only = is_repo_params-master_lang_only ).
 
+    " Make sure there're no leftovers from previous repos
+    ro_repo->rebuild_local_checksums( ).
```

Comment:
Mmm, a new repo is empty ... is it needed ?

Reply:
The rebuild is for local objects which will exist if the package for the repo already contains objects. It was there before, so I kept it.

Reply:
It rebuilds for local objects, which could exist already. But there's most likely a serialize step triggered afterward that does the checksums anyway